src/App.tsx

```diff
@@ -14,37 +15,97 @@ export const goodsFromServer = [
   'Jam',
   'Garlic',
 ];
+enum SortType {
+  ABC = 'ABC',
+  LENGTH = 'LENGTH',
+}
+
+type Props = {
+  sortField: SortType | '';
+  order: boolean;
+};
+
+function getPreparedGoods(goods: string[], { sortField, order }: Props) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortType.ABC:
+          return good1.localeCompare(good2);
+        case SortType.LENGTH:
+          return good1.length - good2.length;
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (order) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType | ''>('');
```

Comment:
```suggestion
  const [sortField, setSortField] = useState<SortType>(SortType.DEFAULT);
```
create default field in enum